### ISSUE TITLE
accuracy of calculated agent-travel distance (and time)?

### ISSUE THREAD

**Issue by yosimor** [jpeterbaker/maxfield#39]
If I understand correctly, the distance between a given pair of portals (which indirectly also determines the amount of required travel time) is calculated based upon the **simple line-of-sight route**, and does not take into account the **real-world walking/driving route**.

If it were somehow possible to utilize the **Google Maps knowledge** of the latter [via the [Distance Matrix API](https://developers.google.com/maps/documentation/distance-matrix/)?] in order to make the calculations more meaningful, that would be most wonderful!

Also, it would be nice if each step of the Agent's plan, in traveling from one link-origin to the next, could show the calculated travel distance (and time) from the previous step.

**Comment by jpeterbaker** [comment]
You are correct about how distances are calculated. I'm not nearly good enough of a programmer to incorporate Google Maps info, but printing travel distance/time to the next portal is certainly possible. I haven't been actively developing this for a while, but I'll leave this on my list of ideas for future improvements.
